features/step_definitions/user.rb

```diff
@@ -72,3 +72,48 @@
 Then(/^(.*) should not be able to sign up$/) do |username|
   expect(user_named(username).persisted?).to be_falsey
 end
+
+
```

Comment:
Space — the final frontier. These are the voyages of the starship `develop`. Its ongoing mission — to program strange new apps — to seek out new smells and new optimizations — to boldly code where no man has coded before.